ui/src/components/userProfile/UserProfileDesktop.js

```diff
@@ -0,0 +1,577 @@
+                    />
+                  </FormControl>
+                  <FormControl>
+                    <FormLabel color="#2C5282">Phone Number</FormLabel>
+                    <Input
+                      size="sm"
+                      borderRadius="5"
+                      {...register("phoneNumber", { required: true })}
+                      isInvalid={errors["phoneNumber"]}
+                      placeholder="Please fill in this field"
+                    />
+                  </FormControl>
+                </Stack>
+                <Stack direction={["column", "row"]} spacing="7%">
+                  <FormControl>
+                    <FormLabel color="#2C5282">
+                      Address: Street Address
+                    </FormLabel>
+                    <Input
+                      size="sm"
+                      borderRadius="5"
+                      {...register("addressStreet")}
+                    />
+                  </FormControl>
+                  <FormControl>
+                    <FormLabel color="#2C5282">Birthday</FormLabel>
+                    <Input
+                      size="sm"
+                      type="date"
+                      borderRadius="5"
+                      {...register("birthday")}
+                    />
+                  </FormControl>
+                </Stack>
+                <Stack direction={["column", "row"]} spacing="7%">
+                  <FormControl flex={1}>
+                    <FormLabel color="#2C5282">Address: District</FormLabel>
+                    <Select
+                      size="sm"
+                      borderRadius="5"
+                      {...register("addressDistrict")}
+                    >
+                      {districtList.map((item) => {
+                        return <option key={"di" + item}>{item}</option>;
+                      })}
+                    </Select>
+                  </FormControl>
+                  <Box flex={1}></Box>
+                </Stack>
+                <Stack direction={["column", "row"]} spacing="7%">
+                  <FormControl flex={1}>
+                    <FormLabel color="#2C5282">Address: Region</FormLabel>
+                    <Select
+                      size="sm"
+                      borderRadius="5"
+                      {...register("addressRegion")}
+                    >
+                      {regionList.map((item) => {
+                        return <option key={"re" + item}>{item}</option>;
+                      })}
+                    </Select>
+                  </FormControl>
+                  <Box flex={1}></Box>
+                </Stack>
+              </Stack>
+              <Button
+                size="sm"
+                mt="5%"
+                color="#0628A3"
+                borderColor="#0628A3"
+                borderRadius="10"
+                variant="outline"
+                type="submit"
+              >
+                Edit Information
```

Comment:
Save instaed of Edit Information?

Reply:
im following the UI designer on this, but that makes more sense to me too, will change that